.travis.yml

```diff
@@ -1,7 +1,7 @@
 # This file has been generated by `make_travis_yml_2.hs`
 # see https://github.com/hvr/multi-ghc-travis for more information
 language: c
-sudo: false
+sudo: true
```

Comment:
Please leave sudo disabled. Otherwise caching doesn’t work and LLVM is built on every commit. Take a look at how GHC is installed in the travis config. You can just add happy there.